docs/overview/trustbloc-did-method.md

```diff
@@ -193,53 +263,151 @@ In some use cases, you might choose never to automatically update to the latest
+    "policy": {"cache": {"max-age": 604800}},
+    "endpoints": [
+        "http://endpoints.stakeholder.two/peer1/",
+        "http://endpoints.stakeholder.two/peer2/",
+        "http://endpoints.stakeholder.two/peer3/"
+    ]
+}
+```
 
-An *Update* operation provides a patch to add or remove keys and service endpoints to a particular DID document.
+In `stakeholder.three/.well-known/did-bloc/stakeholder.three.json` is a JWS signed by `s3VERKEY123456789` wrapping the following:
+```json
+{
+    "domain": "stakeholder.three",
+    "did": "did:trustbloc:consortium.net:s3did12345",
+    "policy": {"cache": {"max-age": 604800}},
+    "endpoints": [
+        "http://endpoints.stakeholder.three/peer1/"
+    ]
+}
+```
 
-##### Recover
+- Parse stakeholder configs
+- Verify stakeholder signatures
+- Pick out several endpoints
+- use each stakeholder's endpoints to fetch the stakeholder's DID doc
+- verify stakeholder key in consortium config is a verification key in the did doc
+- having verified, use endpoints from N stakeholders for a query
 
-A *Recover* operation replaces a DID document for a given DID with a new version of the document, in a way that ensures a stolen device cannot update the document using old operation passwords. This therefore recovers control of the DID document.
+##### Resolver resolves DID using Automatic Bootstrapping
+- list example files
+- summarize automatic bootstrapping process, link to [automatic bootstrapping](#automatic-bootstrapping)
 
-##### Revoke (Delete)
+##### Resolver updates a cached configuration, to perform DID operation
+- list example files
+- including history files
+- Resolver resolves DID from cached consortium, updating to latest
 
-A *Revoke* operation deletes the DID document for a given DID, leaving it as a historical artifact that cannot be accessed through the sidetree endpoints.
 
-### Implementation Notes
+### ---
+TODO: Example server-side user flows? for example:
+- Updating a consortium config and getting it endorsed by a quorum of stakeholders
 
-When implementing a client which performs DID operations in a consortium, it is useful to split out the *read* operation (Resolve) from the *edit* operations (Create, Update, Recover, Revoke). A resolver does not need any special permissions, and resolution is the most common operation, being needed for any transaction using a DID as an identifier.
+### Error Cases
 
-The edit operations require a client which can securely store key material - it must sign the contents of edit operation messages, and must generate and store one-time passwords for later provision, to prove ownership of the DID.
+Error cases which terminate the discovery process in a failure state:
+- Consortium config unavailable: The consortium domain points to a server that isn't functional
+- Insufficient endorsement: Insufficient stakeholders endorse the consortium
+
+Error cases for a specific stakeholder, which can be sidestepped if other stakeholders are available:
```

Comment:
if sufficient other members are available.

Reply:
sidestepped might be an unknown term. you are probably thinking of a stronger form of ignore...